### ISSUE TITLE
Waiting 30 seconds for device /dev/disk/by-label/KAOS_20170217

### ISSUE THREAD

**Issue by 81909ce2298f99ddafa37936ba43eaac** [KaOSx/calamares#77]
#### Submission type

  - [x ] Bug report
  - [ ] Feature Request

#### KaOS ISO date you are having issues with or have a request for

> 20170217

#### How did you create the Live Media?

> GnomeBaker

#### Did you check the md5sum of the downloaded ISO?

> yes

#### Describe the issue you encountered

> After booting from the iso image the start of the live system failed. A timeout occurred while waiting for device /dev/disk/by-label/KAOS_20170217. The dvd has the correct label.

I took  a look at the file system and there wasn't a directory "/dev/disk/by-label/". I was able to fix the problem by entering the following commands:

1. mkdir /dev/disk/by-label/

2. ln -s /dev/sr0 /dev/disk/by-label/KAOS_20170217

3. exit (Shell)

My system is a Levono ThinkCentre M55 PC with Intel Core 2 (Socket LGA775). Hard disk drive and DVD have an SATA bus type.

#### Steps to reproduce the problem

> Just try to install KaOS from the iso image on the system mentioned above with an empty hard disk drive.

#### Include the installation.log:
Sorry, I don't have a log file.

**Comment by demmm** [comment]
Make sure to read what the supported ways are to create live media.  Your method is not.

**Comment by 81909ce2298f99ddafa37936ba43eaac** [comment]
Can you please tell me why do you think that the method of burning an iso image can cause the bug I described above? The dvd **is** bootable and has the correct label. Moreover please take a look at the solution.

**Comment by demmm** [comment]
If written to DVD then failed to mention what write speed used.  If burned with supported software and correct speed then ofc no workaround needed. That you did need to do so shows the media is broken.

**Comment by 81909ce2298f99ddafa37936ba43eaac** [comment]
I wrote the DVD with 2x speed :-)

**Comment by 81909ce2298f99ddafa37936ba43eaac** [comment]
I checked the DVD: 

> $diff /dev/sr0 ~/Downloads/KaOS-2017.02-x86_64.iso
> $

To be on the safe side: 

>$ cat ~/Downloads/KaOS-2017.02-x86_64.md5sum (from your website)
> a9533c7a0ec43f1be5303a6ab8dedf43

>$ md5sum /dev/sr0
> a9533c7a0ec43f1be5303a6ab8dedf43  /dev/sr0

The media isn't broken.

**Comment by demmm** [comment]
The media is clearly broken, needing a hack to make it boot shows.  Please stop trying to justify your hacks and instead start with a normal booting Live media.
GnomeMaker is abandoned for 10 years, so ofc is not on anyone's list of supported media creation.

**Comment by 81909ce2298f99ddafa37936ba43eaac** [comment]
Are you kidding me? I'm not justifying anything and I don't need a hack to boot from the DVD media. I need a hack to start the live system. But this is not the point. You're saying the DVD media is broken. I did my best to prove that's not broken. I used the best method I know to prove it. The md5sums are identical. Hence its proved from my point of view. You still say its broken, but you don't furnish any argument or explanation except that the I used a old software for creating the dvd media. BTW: The software may be not on your list of supported media creation, but this doesn't mean its not on the list of other people. Gnomebaker is still supported in Fedora Linux. You may take a look at "https://rpmfind.net/linux/rpm2html/search.php?query=gnomebaker". I'm using Fedora.

Maybe gnomebaker has bugs, but a bug doesn't explain identical md5sum to me. I would like to understand what's going wrong.

**Comment by 81909ce2298f99ddafa37936ba43eaac** [comment]
One general remark:
I run into 3 traps with my first installation and I solved the problems which occurred. I like KaOS. That is reason for my bug reports. I don't want to waste your time. Its more time consuming to write a bug report than simply to ignore a problem which has been solved for my own purposes.

**Comment by 81909ce2298f99ddafa37936ba43eaac** [comment]
I did some additional testing and the things are getting wired :-) 

First, I burned a new DVD media from my validated ISO image. This time I used "cdrecord": 

`cdrecord -v dev=1,0,0 -speed 2 driveropts=burnfree fs=32m -eject ~/Downloads/KaOS-2017.02-x86_64.iso`

Again, md5sum of the media shows a valid checksum:

> $ md5sum /dev/sr0
> a9533c7a0ec43f1be5303a6ab8dedf43  /dev/sr0
> $

I don't want to erase my previously created installation of KaOS. Hence I used another hard disk. This time I used a SATA SSD (128 GB) instead of simple SATA hard disk (160 GB) as before. 

Interim result: With this new media I was able to start the KaOS live system without any hacks. I tried this with and without partitions on my hard disk. 

Now i tried to boot from my old DVD media which was the basis of my bug report. Same result: I was able to start the KaOS live system without any hacks. 

Bottom line: I'm still sure that generating checksums as described above is a valid method to prove whether a dvd media made out of an iso image is broken or not. But for an unknown reason I can't reproduce the problem with slightly different hardware at moment.
